src/main/java/duke/task/Deadline.java

```diff
@@ -0,0 +1,53 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * Represents a deadline task.
+ */
+public class Deadline extends Task {
+    public static final String IDENTIFIER = "D";
+    private static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("d/MM/yyyy HHmm");
+    private static final DateTimeFormatter DISPLAY_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy hh:mm a");
+
+    private LocalDateTime by;
```

Comment:
While this is a private variable, maybe a more descriptive name would help. That being said, this short name is pretty elegant too, but personally I would optimise for readability and use a longer name.